2021-AUG/src/Footer.tsx

```diff
@@ -26,8 +26,15 @@ const Footer = () => {
               Created by the Computer Science graduating class of 2021
             </Typography>
             <Typography>
-              &copy; {new Date().getFullYear()}: Camosun College
+              &copy; 2021 Camosun College
             </Typography>
+            <Button
+              variant="contained"
+              color="secondary"
+              href="https://capstone.camosun.bc.ca/"
+            >
+              Capstone Home
+            </Button>
```

Comment:
```suggestion
            <Button
              variant="contained"
              color="secondary"
              style={{marginBottom: '5px'}}
              href="https://capstone.camosun.bc.ca/"
            >
              Capstone Home
            </Button>
```

Adds 5 pixels below the Capstone Home button